python-main.js

```diff
@@ -643,6 +654,11 @@ function web_editor(config) {
             // Stop immediate closure
             e.stopImmediatePropagation();
         });
+
+        window.addEventListener('resize', function(){
+            formatHelpPanel();
+        });
```

Comment:
One last thing, and only apply if you like, as it is not going to make much of a difference, inside this function you can check visibility before executing `formatHelpPanel()`.